x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -62,17 +77,17 @@ function useServicesFetcher() {
         }).then((mainStatisticsData) => {
           return {
             requestId: uuid(),
-            mainStatisticsData,
+            ...mainStatisticsData,
           };
         });
       }
     },
     [environment, kuery, start, end]
   );
 
-  const { mainStatisticsData, requestId } = data;
+  const { data: mainStatisticsData = initialData } = mainStatisticsFetch;
```

Comment:
nit: Probably just me being old school but I prefer this

```suggestion
  const mainStatisticsData = mainStatisticsFetch.data ?? initialData;
```